gsoc/2017.md

```diff
@@ -284,7 +284,7 @@ Savina suite was ported to the Reactors framework. To make Reactors
 fully compliant with Savina, the goal of this project is to implement
 the remaining 21 Savina benchmarks using the Reactors framework.
 
-[Savina Benchmark Suite](https://pdfs.semanticscholar.org/b346/9783325ea82d1ded949804a0676acb125484.pdf)
+Savina Benchmark Suite: https pdfs.semanticscholar.org/b346/9783325ea82d1ded949804a0676acb125484.pdf (dead link)
```

Comment:
maybe change the link to https://github.com/shamsimam/savina ?

Reply:
done